src/web/api/src/main/java/org/locationtech/geogig/web/api/CommandBuilder.java

```diff
@@ -56,7 +56,7 @@
  */
 public class CommandBuilder {
 
-    private final static Map<String, Function<ParameterSet, WebAPICommand>> MAPPINGS =
+    private final static Map<String, Supplier<WebAPICommand>> MAPPINGS =
```

Comment:
The map is created with a starting size of 30, but we add 36 mappings, so it would be better to increase the start size to 36 so the map doesn't have to grow on startup. Trivial, but still.